en/docs/streaming/extracting-data-from-static-sources-in-real-time/extracting-data-from-static-sources-in-real-time.md

```diff
@@ -496,4 +516,23 @@ The following is a list of cloud platforms from which you can extract stored dat
 | AWS Simple Cloud Storage (S3) | [S3](https://siddhi-io.github.io/siddhi-io-s3/api/latest/)                                            |
 | Google Cloud Storage          | [GCS](https://siddhi-io.github.io/siddhi-io-gcs/api/latest/)                                          |
 | CosmosDB                      | [CosmosDB](https://github.com/wso2-extensions/siddhi-store-cosmosdb/blob/master/docs/api/latest.md)   |
-| Azure Data Lake               | [azuredatalake](https://siddhi-io.github.io/siddhi-io-azuredatalake/api/latest/#source)               |
\ No newline at end of file
+| Azure Data Lake               | [azuredatalake](https://siddhi-io.github.io/siddhi-io-azuredatalake/api/latest/#source)               |
+
+### Supported mappers
+
+Mappers determine the format in which the event is received. For information about transforming events by changing the format in which the data is received/published, see [Processing Data - Transforming Data](processing-data.md#transforming-data).
```

Comment:
@RukshiW , does the following link actually work? I recall when links used to have the .md it the link used to break. Not sure how this will work.

[Processing Data - Transforming Data](processing-data.md#transforming-data)